cli/validation/index.md

```diff
@@ -164,6 +182,25 @@ We won't show the output here as it's quite long. The lack of any `-f` or
 Profile, meaning it's equivalent to `-f 0` or `--flavour 0`,
 [see automatic profile selection above](#auto-profile).
 
+### Disable error messages
+By default, verapdf contains detailed error messages for each error case in report. The user can disable these messages to speed up the validation by using option `--disableerrormessages`.
+
+### Show progress
+
+User can see the current status of the validation job in console by using option `--progress`.
+
+<a name="logs-customising"></a>Customising logs
+-------------------------------------------------------------------------
+
+User can add logs in report by using option `--addlogs`. Also you can set level of displayed logs by using option `--loglevel`.
+
+<a name="custom-profile"></a>Choose custom profile
+------------------------------------------------
+
+User can validate PDF files according custom profile by using option `--profile` or `-p`. For example
```

Comment:
The user can validate PDF files against a custom validation profile by using option `--profile` or `-p`. For example